docs/conversion/sakai_21_3-21_4_oracle_conversion.sql

```diff
@@ -0,0 +1,6 @@
+-- SAK-47291
+UPDATE gb_gradable_object_t SET EXTERNAL_APP_NAME = 'sakai.assignment.grades' WHERE EXTERNAL_ID LIKE '/assignment/a/%';
+UPDATE gb_gradable_object_t SET EXTERNAL_APP_NAME = 'sakai.lessonbuildertool' WHERE EXTERNAL_ID LIKE 'lesson-builder:%';
+UPDATE gb_gradable_object_t SET EXTERNAL_APP_NAME = 'sakai.attendance' WHERE EXTERNAL_ID LIKE 'sakai.attendance.%';
+UPDATE gb_gradable_object_t SET EXTERNAL_APP_NAME = 'sakai.samigo' WHERE REGEXP_LIKE(EXTERNAL_ID, '^[0-9]+$');
```

Comment:
I'm not used to use oracle, do you think whis query is valid? The intention is to find all the rows containing numbers. These rows will be the ones that are refering to a samigo gradebook item.

Reply:
Thanks @bjones86